fitting/views.py

```
@@ -238,6 +241,46 @@ def post(self, request):
+
+        response = model.generate_content(contents, generation_config={"max_output_tokens": 60})
+        # Gemini 응답 형식 대응
+        analysis = (
+            response.text.strip()
+            if hasattr(response, "text")
+            else response.candidates[0].content.parts[0].text.strip()
+        )
+        # 혹시 모를 길이 초과 방지
+        return analysis[:120]
+
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)